package.json

```diff
@@ -33,8 +33,15 @@
     "yosay": "^2.0.2"
   },
   "devDependencies": {
+    "@k8slens/extensions": "^4.0.0-beta.3",
```

Comment:
Probably makes sense bump to beta.4 (it's already in build pipeline)?

Reply:
fixed in https://github.com/lensapp/generator-lens-ext/pull/11/commits/aee6a269d01bd898a8a8b7225c4a2e721a87e167